src/components/Popups/PopupCursor.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import PopupBase from './PopupBase';
+import { Options } from './Popper';
+import './PopupCursor.scss';
+
+export type Props = {
+    isActive: boolean;
+};
+
+const options: Partial<Options> = {
+    modifiers: [
+        {
+            name: 'offset',
+            options: {
+                offset: [0, 20],
+            },
+        },
+    ],
+    placement: 'right',
+};
+
+const generateGetBoundingClientRect = (x = 0, y = 0) => () => ({
```

Comment:
Does `new DOMRect` work for our purposes?

Reply:
`DOMRect` is not defined in test. It took me one hour to try mocking it but the `new` stuff is complicated. So I changed it back.